### PR TITLE
docs: new typography section - new table

### PR DESCRIPTION
**Pull request by fede-dp**
## Description
I'm updating the typography section in the dialtone documentation, i implemented a table with all the variables.

## Pull Request Checklist

 - [ ] Ask the contributors if a similar effort is already in process or has been solved.
 - [ ] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [ ] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

## Obligatory GIF (super important!)

not sure how the gif works but this is my first one haha

### CODE REVIEW

**Comment by juliodialpad** [comment]
Is this ready for review? you didn't include a description. If it's not ready for review, change it to draft please.

**Comment by fede-dp** [comment]
> Is this ready for review? you didn't include a description. If it's not ready for review, change it to draft please.

Sorry about that! was trying to work in that with @iropolo and try a bunch of times and forget to update the last one, here we go again!

File: apps/dialtone-documentation/docs/design/typography/index.md [hunk 1]
```diff
@@ -1,89 +1,729 @@
...
-  <div class="d-notice__icon">
-    <dt-icon name="info"></dt-icon>
-  </div>
-  <div class="d-notice__content d-stack4">
-    <p class="d-notice__message">
-      <strong>WIP</strong>
-    </p>
-  </div>
-</aside>
+<svg width="746" height="301" viewBox="0 0 746 301" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/design/typography/index.md]: Can we isolate this svg, and other large svgs in it's own file? It's reallllly big. See `SvgLoader.vue`

File: apps/dialtone-documentation/docs/design/typography/index.md [hunk 2]
```diff
@@ -1,89 +1,729 @@
...
+
+Typography is more than just selecting fonts; it's about creating a harmonious and effective reading experience. In this documentation, we'll explore how our chosen typefaces, styles, and layouts come together to convey a consistent brand identity across digital interfaces.
+
+## Core Principles
+
+Understanding the fundamentals is key. Learn about the anatomy of type, the interplay of typefaces, and how to strike the right balance between readability and aesthetics. Discover how size, spacing, and hierarchy contribute to a seamless user experience.
+
+## Typeface selection
+
+To define the Typeface selection, we take the accessibility is a priority to us, so we approach it by using the default system fonts by each operating system to have the best conversion to all possible variables of our product (Japanese, Russian, etc)
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/design/typography/index.md]: "we take the accessibility is a priority to us" Something wrong with the wording here.. Did we mean to say "we make accessibility a priority"?

File: apps/dialtone-documentation/docs/design/typography/index.md [hunk 3]
```diff
@@ -1,89 +1,729 @@
...
+  </tbody>
+</table>
+
+## Responsive & Mobile Typography
+
+In a world of diverse devices, responsive design is paramount. Learn how our typography adapts to different screen sizes and orientations, ensuring a consistent and enjoyable experience across desktops, tablets, and smartphones.
+
+## Tokens
+
+We have our own sections for the Tokens and theming, you can check our typography tokens in the design tokens section
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/design/typography/index.md]: Link to it?

File: apps/dialtone-documentation/docs/design/typography/index.md (hunk 2 above)
Comment (fede-dp) [reply apps/dialtone-documentation/docs/design/typography/index.md]: great! i think the autocorrect somehow change 'make' to 'take', but I like your approach, i think is more straight forward, updating that!

**Comment by iropolo** [comment]
Ping to update @fede-dp

**Comment by francisrupert** [comment]
@fede-dp, I'm a little confused: will this be reconciled with the branch I shared with you the other day?

@fede-dp, @ninamarina, 

Can you break out the Typography sub-types, such that they are they're own table with a third-level headline. 

Both visually and structurally this is awkward: `<tr><td colspan="5">Headers</td></tr>`

It should be something like the attached...

image

**Comment by fede-dp** [comment]
Hey @francisrupert , yes I agree, i was trying to keep it compact, we're talkling with @ninamarina about the table and basically since we don't have more complex tokens, i think the best option is to link to the [tokens](https://dialtone.dialpad.com/tokens/typography.html) section and then the [classes](http://dialtone.dialpad.com/utilities/typography/styles.html#headlines) section, eventually we will need to work on 2nd level tokens for typography and then i think will makes sense to add this to the documentation, what do you think?

**Comment by francisrupert** [comment]
Don't worry about listing the *Design Tokens* on this page. Refer back to the branch I provided you ([screenshot](https://lh3.googleusercontent.com/oiCnvk5fFrCdqx3HKDt0xYFGyzlBY2npmaGQHS-yjvwL3LSsHVDjGajXj6CuCNlDhpHPqNPUnKX0tYwIxOwkwFkrIbIN8NEaFQ=s1600)) and do a simple visual for each of Font, Typescale, Font weight, and Line height. 

The **Styles** definitely should be listed on this page, paired with a simple "Usage" example (linked screenshot above), but don't need to break them down by Design Token.

Ship that and you can build it up afterward, i.e. your _"...eventually we will need to work on 2nd level tokens for typography and then i think will makes sense to add this to the documentation..."_.

**Review by francisrupert — CHANGES_REQUESTED** [review]
Looking good. This'll be a nice little mvp state of this page and we'll build up in future tasks. It's mostly there, some simple fixes attached below:
image

image

image

image

image

image

image

image

**Comment by fede-dp** [comment]
hey! fix everything but the table order, @ninamarina can you help me with that one?

File: apps/dialtone-documentation/docs/design/typography/index.md [hunk 4]
```diff
@@ -1,89 +1,219 @@
...
+      </div>
+      <div><p :class="varName">{{ example }}</p></div>
+    </template>
   </div>
-</aside>
+</code-well-header>
+
+## Responsive & Mobile Typography
+
+In a world of diverse devices, responsive design is paramount. Learn how our typography adapts to different screen sizes and orientations, ensuring a consistent and enjoyable experience across desktops, tablets, and smartphones.
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/design/typography/index.md]: Remove this.

**Review by francisrupert — APPROVED** [review]
Approved, with one tiny change noted below. Ship it!

**Pull request merged**